docs/config.md

```diff
@@ -96,271 +76,149 @@ openstack security group rule create --egress kubernetes
-NAME                     STATUS   ROLES    AGE    VERSION
-openstack-master-g8vrl   Ready    master   112m   v1.15.0
-openstack-node-6b2v7     Ready    <none>   109m   v1.15.0
-```
-
-Then find file at `examples/openstack/machine-deployment.yaml.template` and do modification according to your settings,
-In below example, the machine replics are set to 2:
-```
-# kubectl --kubeconfig kubeconfig apply -f examples/openstack/out/machinedeploy.yaml
-machinedeployment.cluster.k8s.io/test1-machinedeployment created
-
-# kubectl --kubeconfig kubeconfig get machinedeployment
-NAME                      AGE
-test1-machinedeployment   45s
 
-# kubectl --kubeconfig kubeconfig get machinesets
-NAME                                 AGE
-test1-machinedeployment-8676464b5d   50s
-```
-
-Wait a while and you will see machines and nodes are created:
-```
-kubectl --kubeconfig kubeconfig get machines
-NAME                                       PROVIDERID                                           PHASE
-openstack-master-g8vrl                     openstack:////a87b5caf-8fc5-4943-b0b3-e8743c138c64
-openstack-node-6b2v7                       openstack:////6c8efe5f-e993-430e-a60b-f20f29560240
-test1-machinedeployment-8676464b5d-24fv4   openstack:////c886b36a-9657-40cc-a81f-bffcc73501a8
-test1-machinedeployment-8676464b5d-67jsw   openstack:////a6b14f22-9b87-48c1-a7de-7ccaea6f3b81
-
-# kubectl --kubeconfig kubeconfig get nodes
-NAME                                       STATUS   ROLES    AGE     VERSION
-openstack-master-g8vrl                     Ready    master   145m    v1.15.0
-openstack-node-6b2v7                       Ready    <none>   142m    v1.15.0
-test1-machinedeployment-8676464b5d-24fv4   Ready    <none>   3m25s   v1.15.0
-test1-machinedeployment-8676464b5d-67jsw   Ready    <none>   4m11s   v1.15.0
-```
-
-The new 2 machines with test1-machinedeployment- are newly created.
-
-Use following command to delete the machinedeployment:
-```
-# kubectl --kubeconfig kubeconfig delete machinedeployments test1-machinedeployment
-machinedeployment.cluster.k8s.io "test1-machinedeployment" deleted
-
-# kubectl --kubeconfig kubeconfig get machines
-NAME                     PROVIDERID                                           PHASE
-openstack-master-g8vrl   openstack:////a87b5caf-8fc5-4943-b0b3-e8743c138c64
-openstack-node-6b2v7     openstack:////6c8efe5f-e993-430e-a60b-f20f29560240
-```
+If creating servers in your OpenStack takes a long time, you can increase the timeout, by default it's 5 minutes. You can set it via the `CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT` in your Cluster API Provider OpenStack controller deployment.
 
-#### Custom pod network CIDR
+## Custom pod network CIDR
 
 If `192.168.0.0/16` is already in use within your network, you must select a different pod network CIDR. You have to adjust the CIDR `192.168.0.0/16` with your own in:
-* [examples/addons.yaml](../examples/addons.yaml) 
+* [examples/_out/addons.yaml](../examples/_out/addons.yaml) 
```

Comment:
_out/addons.yaml is not in repository so that you can not link to that.

Reply:
I added a comment that it's about the generated example files. The whole doc is meant as configuration guide based on the generated examples

Reply:
I understand this is about generated examples. But showing 404 page once we click the link is not good.
I think it is better to show `examples/_out/addons.yaml`.

Reply:
I could also just write addons.yaml like I did e.g. with cluster.yaml without a link

Reply:
I adjusted it as proposed